src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/NavLink.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public class NavLink
+    {
+        public NavLink()
+        {
+        }
+
+        public NavLink(
```

Comment:
I think it's not required to define a constructor just to pass the property values.
the consumer can simply use the property initializer as below:
```cs
new NavLink { Name = "test", Key = "key" }
```